lms/templates/lms-deployment.yaml

```diff
@@ -89,6 +89,10 @@ spec:
           {{- toYaml . | nindent 8 }}
         {{- end }}
       {{- end}}
+      initContainers:
+      - name: wait-for-orchestrator
+        image: alpine/curl:latest
+        command: ['sh', '-c', 'until curl -s http://{{ include "molt-lms.fullname" . }}-orchestrator:{{ .Values.orchestrator.service.metricsPort }}/healthz; do echo waiting for orchestrator; sleep 2; done;']
```

Comment:
nit: is it possible to emit log (something like `LMS waiting for orchestrator to be initiated`) in each iteration?

Reply:
Yup! Let me update the log to be more explicit.